crates/biome_test_utils/src/lib.rs

```diff
@@ -204,13 +204,25 @@ pub fn code_fix_to_string<L: ServiceLanguage>(source: &str, action: AnalyzerActi
 /// corresponding to the directory name. E.g., `style/useWhile/test.js`
 /// will be analyzed with just the `style/useWhile` rule.
 pub fn parse_test_path(file: &Path) -> (&str, &str) {
-    let rule_folder = file.parent().unwrap();
-    let rule_name = rule_folder.file_name().unwrap();
+    let mut root_found = false;
+    let mut group_name = "";
+    let mut rule_name = "";
+
+    for component in file.iter().rev() {
+        if component == "specs" || component == "suppression" {
+            root_found = true;
+            break;
+        }
+
+        rule_name = group_name;
+        group_name = component.to_str().unwrap_or_default();
+    }
 
-    let group_folder = rule_folder.parent().unwrap();
-    let group_name = group_folder.file_name().unwrap();
+    if !root_found {
+        panic!("Failed to find group and rule");
```

Comment:
Can you provide a better message? This is mostly for us developers, so if this panics, we should provide a message that tells the developer how to fix the issue.